lib/parser.js

```diff
@@ -15,7 +15,7 @@ module.exports.parse = parse;
 module.exports.parseFromUrl = parseFromUrl;
 module.exports.registerSchemaParser = registerSchemaParser;
 
-async function parse(asyncapiYAMLorJSON, options = {}) {
+async function parse(asyncapiYAMLorJSON, options) {
```

Comment:
This should default to `{}`. Check out the original (updated) branch. It checks for `options.path` instead of simply checking for `options`. There's now a unit test covering this case.